build.sbt

```diff
@@ -1253,6 +1254,20 @@ lazy val nettyServerCats: ProjectMatrix = (projectMatrix in file("server/netty-s
   .jvmPlatform(scalaVersions = scala2And3Versions)
   .dependsOn(serverCore, nettyServer, cats, serverTests % Test)
 
+lazy val nettyServerZio: ProjectMatrix = (projectMatrix in file("server/netty-server/zio"))
+  .settings(commonJvmSettings)
+  .settings(
+    name := "tapir-netty-server-zio",
+    libraryDependencies ++= Seq(
+      "io.netty" % "netty-all" % "4.1.82.Final",
```

Comment:
let's extract netty's version to a constant in the build so that it's not repeated